
We do not perform DESTDIR based packaging - so we do not care about such files ...

  - Rene Rebe <rene@exactcode.de>

--- ./bin/package-ooo.vanilla	2005-04-19 10:12:23.000000000 +0200
+++ ./bin/package-ooo	2005-04-19 10:13:59.000000000 +0200
@@ -273,11 +273,6 @@
     mv $1.unique $1
 }
 
-if ! test -f $OODESTDIR/gid_Module_Root; then
-    echo "Failed to generate package file lists";
-    exit 1;
-fi
-
 cd $OODESTDIR
 
 if test -f gid_Module_Root_Files_2; then
